src/diffusers/pipelines/ledits_pp/ledits_utils.py

```diff
@@ -0,0 +1,184 @@
+import math
+from typing import List, Tuple, Union
+
+import numpy as np
+import torch
+import torch.nn.functional as F
+from PIL import Image
+
+from ...image_processor import PipelineImageInput
+
+
+def load_images(
```

Comment:
Do we really need this method? Can't we try to re-use:
https://github.com/huggingface/diffusers/blob/8842bcadb94bd4a84481ac727aff97cd901e5024/src/diffusers/utils/testing_utils.py#L369
and 
https://github.com/huggingface/diffusers/blob/main/src/diffusers/image_processor.py
?

Reply:
I don't think we can use the existing image processor without making some changes. 
Mainly because we would like the image resizing to be somewhat flexible. We would like to automatically downsize to the default resolution (512px) but still allow for non-square images to be edited. 
I think this should be possible with the `preprocess` method in https://github.com/huggingface/diffusers/blob/main/src/diffusers/image_processor.py, but only for PIL inputs.